tests/modules/test_services.py

```diff
@@ -89,10 +89,3 @@ def test_receive_and_unpack_on_worker(self):
         result_tree = set(filename[len(destination):] for filename in get_files_recursive(destination))
         original_tree = set(filename[len(source):] for filename in get_files_recursive(source))
         self.assertEqual(result_tree, original_tree)
-
-    def test_no_work_prov(self):
-        obj = Unpacker()
-        obj.engine = EngineEmul()
-        obj.engine.config[Provisioning.PROV] = 'cloud'
-        obj.parameters.merge({Unpacker.FILES: ['notexists.zip']})
-        obj.prepare()
```

Comment:
Yes, I mean a check for any service (unpacker for example). As we hope service startup is conditional, we should check this behaviour. By others words, I think we should cover Service.should_run() with unit test.